tests/kernel/mem_protect/syscalls/prj.conf

```diff
@@ -1,5 +1,5 @@
 CONFIG_ZTEST=y
 CONFIG_TEST_USERSPACE=y
 CONFIG_TIMESLICING=y
-CONFIG_TIMESLICE_SIZE=10
+CONFIG_TIMESLICE_SIZE=20
```

Comment:
Why the change? Working around lingering issues?

Reply:
This is the timeslicing fix.  There were tests that were asking for timeslices so small they couldn't be reliably scheduled by the timer driver, and a combination of recent changes to the timeslice layer (which for subtle reasons dropped integration some code that would elide the z_clock_set_timeout() call on imminent expirations) and more deterministic thread abort behavior meant that we were hitting the hang condition.

But really it's a bug in the tests.  Minimum timeslice size when CONFIG_TICKLESS_KERNEL is two ticks.  One won't work right (and now the kernel protects against it).